src/cli/abi.ts

```diff
@@ -0,0 +1,198 @@
+import ts, { Decorator, NodeArray } from "typescript";
+import * as abi from "near-abi";
+import * as TJS from "typescript-json-schema";
+import { JSONSchema7 } from "json-schema";
+import * as fs from "fs";
+import { LIB_VERSION } from "../version.js";
```

Comment:
It looks like this generated version.js isn't necessary, since you have `parseMetadata` below, which still load package.json?

Reply:
`parseMetadata` actually loads the package file supplied by the user (i.e. contract's package.json). But I also need to parse the near-sdk-js package.json. We can go with what @miraclx suggested in the other thread though if that looks good to you.